_layouts/default.html

```diff
@@ -7,6 +7,10 @@
 {% seo %}
 
     <link rel="stylesheet" href="{{ '/assets/css/style.css?v=' | append: site.github.build_revision | relative_url }}">
+    {% assign favicon = page.favicon | default: site.favicon %}
+    {% if favicon %}
+      <link rel="shortcut icon" href="{{ page.favicon }}">
```

Comment:
We'll want to add `{{ page.favicon | absolute_url }}` to ensure the URL is correct on subpages.